.github/workflows/build.yml

```diff
@@ -69,3 +69,16 @@ jobs:
       -
         name: Run test
         run: vendor/bin/ecs check --config ecs.php tests ecs.php
+
+      -
+        name: "Check exported files"
+        run: |
+            EXPECTED="LICENSE,README.md,composer.json,ecs.php"
+            CURRENT="$(
+                git archive HEAD \
+                    | tar --list \
+                    | paste --serial --delimiters=","
+            )"
+            echo "CURRENT =${CURRENT}"
```

Comment:
```suggestion
            echo "CURRENT=${CURRENT}"
```

Reply:
This would unalign current and expected file list.

Reply:
```
CURRENT =LICENSE,README.md,composer.json,ecs.php
EXPECTED=LICENSE,README.md,composer.json,ecs.php
```